### ISSUE TITLE
Create /browse view to render custom IA query results on OpenLibrary

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#602]
Right now, many of the new carousels on our homepage pull their data from archive.org (because its elasticsearch allows us to query based on only available book). But the links we display (to show more results) take the user to archive.org (not a page on openlibrary.org) which is jarring.

We should use archive.org APIs to make a /browse page which allows users to see more (only available) carousel results from openlibrary. 

e.g. a url like https://openlibrary.org/browser?query=creator:(Dahl,Roald) OR creator:(Clancy,Tom)

We should design this page to be our first truly mobile first experience and see how creative we can be

**Comment by mekarpeles** [comment]
cc: @jeansaung

**Comment by LeadSongDog** [comment]
It's not just jarring, it is outright dysfunctional. Many network policies block access to IA as an attack vector. This means that apparently OL-local links fail. 
e.g. at https://openlibrary.org/works/OL15344038W/L'_economia_delle_citt%C3%A0 "You might also like" links to https://archive.org/search.php?query=collection:(inlibrary)%20AND%20loans__status__status:AVAILABLE%20AND%20openlibrary_work:(*)%20AND%20(subject:%22Industries%22%20OR%20subject:%22Urban%20economics%22%20OR%20subject:%22Social%20aspects%22%20OR%20subject:%22Economic%20development%22%20OR%20subject:%22Social%20aspects%20of%20Industries%22)%20AND%20!openlibrary_work:(OL15344038W)

**Comment by mekarpeles** [comment]
#603 
We may make this a primary focus of the hackathon. We'd love designers to join us for the Hackthon, either remotely or in person in SF - Internet Archive (300 Funston) @ 11am.

Goals:
- a completely mobile responsible page
- uses archive.org advancedsearch API
- displays on OpenLibrary /browse
- books displayed with cover, title, author, borrow button

**Comment by mekarpeles** [comment]
cc @rchrd2, this is related to #630

**Comment by rchrd2** [comment]
Hello. I drew up some rough wireframes of an MVP. It's inspired by SimpleE (https://itunes.apple.com/us/app/simplye/id1046583900?mt=8)

![scan nov 2 2017 5 57 pm page 1](https://user-images.githubusercontent.com/182479/32357195-c50be86e-bff7-11e7-9735-9a2124d66d9f.jpg)
![scan nov 2 2017 5 57 pm page 2](https://user-images.githubusercontent.com/182479/32357200-c79f618c-bff7-11e7-89b2-09ce9ce9bc87.jpg)
![scan nov 2 2017 5 57 pm page 3](https://user-images.githubusercontent.com/182479/32357202-c9ba03be-bff7-11e7-876d-b9c5258043ec.jpg)

The idea is that you can browse, save, and view books, in a mobile-friendly app-like interface.
Login is optional for accessing loan-restricted books.  So I guess, "favoriting" should work in both localstorage and through the open library lists api (want to read).

I suspect that there is a nice mobile webapp framework which can handle a lot of the UI chrome. For bookreader I used http://mmenu.frebsite.nl, but I don't know if it solves all the UI issues here.

**Comment by jdlrobson** [comment]
Is this still a problem? I'm not quite understanding the issue here.  I'm not seeing an links to archive.org

**Comment by guyjeangilles** [comment]
@mekarpeles can you elaborate on Jon's point? The home page doesn't link to the Internet Archive currently. I'm not sure a /browse feature would add much more value.

**Comment by jamesachamp** [comment]
To clarify, most carousel header links ("Books We Love", "Recently Returned", etc) will send patrons to an Internet Archive search results page.  From this search page there doesn't appear to be a path back to Open Library (aside from the browser's back button).  To my mind this is unexpected behavior, and a browse page would improve user experience.

**Comment by Yashs911** [comment]
@jamesachamp If the link opens in a new tab, it might help in improving the user experience

**Comment by tfmorris** [comment]
> To clarify, most carousel header links (...) will send patrons to an Internet Archive search results page. From this search page there doesn't appear to be a path back to Open Library (aside from the browser's back button). To my mind this is unexpected behavior, and a browse page would improve user experience.

Yup, that sucks, as @LeadSongDog noted over 3 years ago -- and even for the individual search results, the link back to OpenLibrary will be "below the fold" (ie collapsed).

Another aspect of IA based searches is that they don't use any OpenLibrary metadata, so, for example, authors are considered the same if they are lexical match rather than if they actually have the same OpenLibrary author record.

**Comment by LeadSongDog** [comment]
Well, to be fair, it has only been unblocked since Apr 2019 when #2070 closed and #2071 merged. Still it would be nice to get this cleared up.

**Comment by lokesh** [comment]
This issue is 8+ years old. I'm assuming the architecture has evolved since then.

Feel free to reopen if still relevant.
